schema/v1_0_0/manifest.schema.json

```
@@ -15,13 +15,25 @@
         "type": "object",
         "required": ["file", "last_updated"],
         "properties": {
-          "file": { "type": "string", "pattern": "^[^:]+$", "description": "Repository-relative path under 'pages/' to the file that defines this page" },
-          "last_updated": { "type": "string", "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}Z$", "description": "Timestamp the page was last updated, in UTC (YYYY-MM-DDThh:mm:ssZ)" },
-          "description": { "type": "string", "description": "Optional human-readable description of the page" }
+          "file": {
+            "type": "string",
+            "pattern": "^(pages/)?[a-z0-9_\\-/]+\\.(wiki|lua)$",
```

```suggestion
            "pattern": "^(pages/)?[a-z0-9_\\-/]+\\.(wiki|lua|js|css)$",
```

may want to package up some js or css too